Update readme (#51)
kc-si/JSCLook: README.md

> # This application displays joint stock companies from Poland. Here you can see data such as stock indices, shareholder list, etc.

Reviewer: I think it would look nicer this way:

```suggestion
# JSCLook 

This application displays joint stock companies from Poland. Here you can see data such as stock indices, shareholder list, etc.
```

##  Currently the site is under construction, but you can see a draft on [jsclook.herokuapp.com](https://jsclook.herokuapp.com/)

App is build on Ruby 3.1.0p0

## Installation:

1. Download repository

> 2. Install gem libraries: to install go to main of this repository and run from command line:

Reviewer: I think you need to add "Install Ruby" before Step 2.

>        `bundle install`

Reviewer: ```suggestion
    ```shell
    gem install bundler
    bundle install
    ```
```

3. App need Postgresql database, you can find it at [https://www.postgresql.org/](https://www.postgresql.org/)

> 4. When you have installed Postgresql then cerate user with password  for app, in Postgresql:

Reviewer: ```suggestion
4. When you have installed Postgresql then create user with password for app, in Postgresql:
```

     `username: jsclook`
     `password: 'jsclookRails'`

     user and password are set in `/config/database.yml`

    Information how to create user in Postgresql you can find by read Postgresql documentation.

5.  Database creation:

    go to main of this repository and run from command line:

        `bin/rails db:migrate`

    app should create database and tables in postgresql.

    To seed databse with sample data:

        `bin/rails db:seed`

>     then app fill database with data from file `db/seeds.rb`

Reviewer: I think `bin/rails db:setup` would handle all of these (you can check it). It wouldn't go thorough the migrations actually (as `bin/rails db:migrate` does), but it would simply create the tables based on `db/schema.rb` which is faster and should work the same (assuming you have properly generated `db/schema.rb` in your repo which should be true).

Author: Correct, `db/schema.rb` is properly generated, so I can use this way

> 6. Now start the serwer by running from main directory:

Reviewer: ```suggestion
6. Now start the server by running the following command from the app's directory:
```

    `bin/rails server`

>     app should start serwer on your local machine,

Reviewer: ```suggestion
    app should start server on your local machine,
```

>     you can accesss site from your browser `http://localhost:3000/`

Reviewer: ```suggestion
    you can access site from your browser http://localhost:3000/.
```

# JSCLook

>  This application displays joint stock companies from Poland. Here you can see data such as stock indices, shareholder list, etc.

Reviewer: ```suggestion
 This application displays joint stock companies from Poland. There you can see data such as stock indices, shareholder list, etc.
```

> ##  Currently the site is under construction, but you can see a draft on [jsclook.herokuapp.com](https://jsclook.herokuapp.com/)

Reviewer: Maybe beta would be nicer?

```suggestion
##  Currently the site is under construction, but you can see a beta version on [jsclook.herokuapp.com](https://jsclook.herokuapp.com/)
```

App is build on Ruby 3.1.0p0

## Installation:

1. Download repository

2. Install Ruby, for more information visit [https://www.ruby-lang.org/](https://www.ruby-lang.org/)

> 3. Install gem libraries. To install go to main directory of this repository and run from command line:

Reviewer: ```suggestion
3. Install gems. Go to main directory of this repository and run from command line:
```

>     If you do not have installed bundler already, install it by run command:

Reviewer: ```suggestion
    If you do not have installed bundler already, install it by running command:
```

        `gem install bundler`

>     then install gem libraries :

Reviewer: ```suggestion
    then install gems:
```

        `bundle install`

> 4. App need Postgresql database, you can find it at [https://www.postgresql.org/](https://www.postgresql.org/)

Reviewer: ```suggestion
4. App needs PostgreSQL database, you can find it at [https://www.postgresql.org/](https://www.postgresql.org/)
```

> 5. When you have installed Postgresql then cerate user with password for app, in Postgresql:

Reviewer: ```suggestion
5. When you have installed Postgresql then create user with password for app, in PostgreSQL:
```

     `username: jsclook`
     `password: 'jsclookRails'`

     user and password are set in `/config/database.yml`

    Information how to create user in Postgresql you can find by read Postgresql documentation.

6.  Database creation:

>     go to main of this repository and run from command line:

Reviewer: ```suggestion
    go to root directory of this repository and run from command line:
```

or

```suggestion
    go to main directory of this repository and run from command line:
```
